packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-link/wallet-link-wrapper.tsx

```diff
@@ -16,58 +18,53 @@ export type TWalletLinkWrapper = {
     }[];
 };
 
-const WalletLinkWrapper = observer(({ wallet_details, account_list }: TWalletLinkWrapper) => {
+const WalletLinkWrapper = ({ wallet_details, account_list }: TWalletLinkWrapper) => {
```

Comment:
@aum-deriv why did you delete observer here?

Reply:
@sergei-deriv nice catch 👍. Will added it back.